web/xfa_layer_builder.css

```diff
@@ -221,6 +221,8 @@
 }
 
 .xfaLink {
+  display: inline-block;
```

Comment:
Yes, as I mentioned I added it because the inline display can't receive any height. When changing the display the link start working and the absolute position it's for the text in the links to show properly.